doc/api/next_api_changes/removals/26965-ER.rst

```diff
@@ -0,0 +1,28 @@
+As part of a `multi-step process
+<https://github.com/matplotlib/matplotlib/issues/20853>`_ we are refactoring
+the global state for managing the registered colormaps.
+
+In Matplotlib 3.5 we added a `.ColormapRegistry` class and exposed an instance
+at the top level as ``matplotlib.colormaps``. The existing top level functions
+in `matplotlib.cm` (``get_cmap``, ``register_cmap``, ``unregister_cmap``) were
+changed to be aliases around the same instance.
+
+In Matplotlib 3.6 we have marked those top level functions as pending
+deprecation with the intention of deprecation in Matplotlib 3.7. The following
+functions have been marked for pending deprecation:
```

Comment:
This paragraph is still written in the tense of the 3.6 release. It should be stating things as they are for 3.9 (i.e., that things are removed.)